docs/people-and-groups/authenticating-with-saml.md

```diff
@@ -8,54 +8,60 @@ redirect_from:
 
 {% include plans-blockquote.html feature="SAML authentication" %}
 
-The open source edition of Metabase includes the option to set up with [Google Sign-in or LDAP](./google-and-ldap.md), but [some plans](https://www.metabase.com/pricing) let you connect your SAML- or [JWT](./authenticating-with-jwt.md)-based SSO. Integrating your SSO with Metabase allows you to:
+Integrating your SSO with Metabase allows you to:
 
-- automatically pass user attributes from your SSO to Metabase in order to power data sandboxes
-- let your users access Metabase without re-authenticating.
+- Automatically pass user attributes from your SSO to Metabase in order to power [data sandboxes](../permissions/data-sandboxes.md).
+- Let people access Metabase without re-authenticating.
 
-## Turning on SAML-based SSO
+## Confirm the password for your Metabase admin account
 
-Before beginning your SAML set-up, make sure you know the password for your Metabase admin account. If anything becomes misconfigured during the set-up process, an "Admin backup login" option on the sign-in screen is available.
+Before setting up SAML, make sure you know the password for your Metabase admin account. If you encounter any issues during the setup process, you can login via the "Admin backup login" option on the sign-in screen.
 
-To get started, head over to the Settings section of the Admin Panel, then click on the **Authentication** tab. Click the **Configure** button in the SAML section of the Authentication page, and you'll see this form:
+## Setting up SAML with your IdP
```

Comment:
```suggestion
## Setting up SAML with your IdP in Metabase
```
there's a bit of ambiguity, like does it refer to setting up SAML _in_ Okta or _using_ Okta _in_ Metabase. This will also match the structure of individual SAML articles better

Reply:
Agree. This is a good note.